content/about/xsf/communication-team.md

```diff
@@ -7,13 +7,26 @@ layout: single_xsf
 ---
 
 The Communication Team's mission is to inform the XMPP community and interested parties on news and recent developments within XMPP ecosystem.
+Furthermore, with communication and expansion of our public relations we also attempt to be an instance of reference in the diverse and decentralized community.
+This should help to growth the network and its application, too.
```

Comment:
These can be removed, they don't add value here.

Reply:
7f075c85a5e32f771d686c8355bf6420587dc1a6

feel free to merge now